src/SSVOpenHexagon/Core/Steam.cpp

```diff
@@ -196,16 +196,16 @@ bool steam_manager::set_rich_presence_in_menu()
 }
 
 bool steam_manager::set_rich_presence_in_game(
-    std::string_view level_name, float time)
+    std::string level_name_format, std::string difficulty_mult_format, std::string time_format)
```

Comment:
AFAICS these could all be `std::string_view` instances, which are very lightweight and cheap to copy. Any reason to make them `std::string` instead? 

```suggestion
    std::string_view level_name_format, std::string_view difficulty_mult_format, std::string_view time_format)
```